plugins/saveactions.c

```diff
@@ -862,11 +1427,159 @@ GtkWidget *plugin_configure(GtkDialog *dialog)
+        gtk_box_pack_start(GTK_BOX(notebook_vbox), inner_vbox, TRUE, TRUE, 5);
+        gtk_notebook_insert_page(GTK_NOTEBOOK(notebook),
+            notebook_vbox, gtk_label_new(_("Untitled Document Save")), NOTEBOOK_PAGE_UNTITLEDDOCUMENTSAVE);
+
+		disabled_radio = gtk_radio_button_new_with_mnemonic(NULL, _("Disabled"));
+		pref_widgets.untitled_document_save_disabled_radio = disabled_radio;
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), disabled_radio);
+		gtk_button_set_focus_on_click(GTK_BUTTON(disabled_radio), FALSE);
+		gtk_container_add(GTK_CONTAINER(inner_vbox), disabled_radio);
+		g_signal_connect(disabled_radio, "toggled",
+			G_CALLBACK(radio_toggled_cb), GINT_TO_POINTER(NOTEBOOK_UNTITLEDDOCUMENTSAVE_RADIO_DISABLED));
+
+		/* Instantsave */
+
+		instantsave_radio = gtk_radio_button_new_with_mnemonic_from_widget(
+			GTK_RADIO_BUTTON(disabled_radio), _("Instant Save"));
+		pref_widgets.untitled_document_save_instantsave_radio = instantsave_radio;
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), instantsave_radio);
+		gtk_button_set_focus_on_click(GTK_BUTTON(instantsave_radio), FALSE);
+		gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(instantsave_radio), enable_instantsave);
+		gtk_container_add(GTK_CONTAINER(inner_vbox), instantsave_radio);
+		g_signal_connect(instantsave_radio, "toggled",
+			G_CALLBACK(radio_toggled_cb), GINT_TO_POINTER(NOTEBOOK_UNTITLEDDOCUMENTSAVE_RADIO_INSTANTSAVE));
+
+		entry_dir_label_text = g_strdup_printf(
+			_("_Directory to save files in (leave empty to use the default: %s):"), g_get_tmp_dir());
+		label = gtk_label_new_with_mnemonic(entry_dir_label_text);
+		gtk_misc_set_alignment(GTK_MISC(label), 0, 0.5);
+		gtk_box_pack_start(GTK_BOX(inner_vbox), label, FALSE, FALSE, 0);
+		g_free(entry_dir_label_text);
+
+		pref_widgets.instantsave_entry_dir = entry_dir = gtk_entry_new();
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), entry_dir);
+		if (!EMPTY(instantsave_target_dir))
+			gtk_entry_set_text(GTK_ENTRY(entry_dir), instantsave_target_dir);
+
+		button = gtk_button_new();
+		g_signal_connect(button, "clicked",
+			G_CALLBACK(target_directory_button_clicked_cb), entry_dir);
+
+		image = gtk_image_new_from_stock(GTK_STOCK_OPEN, GTK_ICON_SIZE_BUTTON);
+		gtk_container_add(GTK_CONTAINER(button), image);
+
+		hbox = gtk_hbox_new(FALSE, 6);
+		gtk_box_pack_start(GTK_BOX(hbox), entry_dir, TRUE, TRUE, 0);
+		gtk_box_pack_start(GTK_BOX(hbox), button, FALSE, FALSE, 0);
+		gtk_box_pack_start(GTK_BOX(inner_vbox), hbox, FALSE, FALSE, 0);
+
+		help_label = gtk_label_new(
+			_("<i>If you set the Instant Save directory to a directory "
+			  "which is not automatically cleared,\nyou will need to cleanup instantly saved files "
+			  "manually. The Instant Save plugin will not delete the created files.</i>"));
+		gtk_label_set_use_markup(GTK_LABEL(help_label), TRUE);
+		gtk_misc_set_alignment(GTK_MISC(help_label), 0, 0.5);
+		gtk_widget_set_margin_bottom(GTK_LABEL(help_label), 8);
```

Comment:
The cast to `GTK_LABEL` is wrong, just remove it.

Reply:
removed